packages/unmock-node/src/__tests__/dynamicService.test.ts

```diff
@@ -314,129 +35,6 @@ describe("Tests dynamic path tests", () => {
-      unmock
-        .nock("https://www.foo.com", "foo")
-        .get("/?q&m=1&") // include an empty query
-        .reply(200);
-      expectNServices(1);
-      expect(getPrivateSchema("foo").paths["/"].parameters[0].in).toEqual(
-        "query",
-      );
-      expect(getPrivateSchema("foo").paths["/"].parameters[0].name).toEqual("");
-      expect(getPrivateSchema("foo").paths["/"].parameters[0].schema).toEqual({
-        type: "null",
-      });
-      expect(getPrivateSchema("foo").paths["/"].parameters[1].in).toEqual(
-        "query",
-      );
-      expect(getPrivateSchema("foo").paths["/"].parameters[1].name).toEqual(
-        "m",
-      );
-      expect(getPrivateSchema("foo").paths["/"].parameters[1].schema).toEqual({
-        type: "string",
-        enum: ["1"],
-      });
-    });
-  });
-
-  describe("Request headers can be specified", () => {
-    it("An empty request header results in a viable spec", () => {
-      expectNServices(0);
-      unmock
-        .nock("https://www.foo.com", {}, "foo")
-        .get("/")
-        .reply(200);
-      expectNServices(1);
-      expect(Object.keys(getPrivateSchema("foo").paths)).toEqual(["/"]);
-    });
-
-    it("A request header is correctly propagated", () => {
-      expectNServices(0);
-      unmock
-        .nock("https://www.foo.com", { reqheaders: { hello: "world" } }, "foo")
-        .get("/")
-        .reply(200);
-      expectNServices(1);
-      expect(getPrivateSchema("foo").paths["/"].parameters[0].in).toEqual(
-        "header",
-      );
-      expect(getPrivateSchema("foo").paths["/"].parameters[0].name).toEqual(
-        "hello",
-      );
-      expect(getPrivateSchema("foo").paths["/"].parameters[0].schema).toEqual({
-        type: "string",
-        enum: ["world"],
-      });
-    });
-  });
-
-  describe("tldr adds 9 levels of arbitrary paths to requests", () => {
-    it("ignores unimportant stuff", () => {
-      expectNServices(0);
-      unmock.nock("https://www.foo.com", "foo").tldr();
-      expectNServices(1);
-      expect(Object.keys(getPrivateSchema("foo").paths).length).toEqual(9);
-    });
-  });
-
-  describe("Reply headers can be specified", () => {
-    it("An empty reply header results in a viable spec", () => {
-      expectNServices(0);
-      unmock
-        .nock("https://www.foo.com", "foo")
-        .get("/")
-        .reply(200, "", {});
-      expectNServices(1);
-      expect(Object.keys(getPrivateSchema("foo").paths)).toEqual(["/"]);
-    });
-
-    it("A reply header is correctly propagated", () => {
-      expectNServices(0);
-      unmock
-        .nock("https://www.foo.com", "foo")
-        .get("/")
-        .reply(200, "", { hello: "world" });
-      expectNServices(1);
-      expect(
-        getPrivateSchema("foo").paths["/"].get.responses["200"].headers.hello
-          .schema,
-      ).toEqual({ type: "string", enum: ["world"] });
-    });
-  });
-
   describe("schema generates valid stuff", () => {
```

Comment:
I know this wasn't part of your changes for this PR, but can we use a more descriptive word than "stuff" here? 😂

Reply:
We should! I fully agree 🙂